src/tableView.js

```diff
@@ -11,671 +11,792 @@ import {
 } from './helpers/dom/element';
 import EventManager from './eventManager';
 import { stopPropagation, isImmediatePropagationStopped, isRightClick, isLeftClick } from './helpers/dom/event';
+import { clearTextSelection } from './helpers/mixed';
 import Walkontable from './3rdparty/walkontable/src';
 import { handleMouseEvent } from './selection/mouseEventHandler';
 
-/**
- * Cross-platform helper to clear text selection.
- */
-const clearTextSelection = function() {
-  // http://stackoverflow.com/questions/3169786/clear-text-selection-with-javascript
-  if (window.getSelection) {
-    if (window.getSelection().empty) { // Chrome
-      window.getSelection().empty();
-    } else if (window.getSelection().removeAllRanges) { // Firefox
-      window.getSelection().removeAllRanges();
-    }
-  } else if (document.selection) { // IE?
-    document.selection.empty();
+const privatePool = new WeakMap();
+class TableView {
+  /**
+   * @param {Hanstontable} instance Instance of {@link Handsontable}
+   */
+  constructor(instance) {
+    /**
+     * Instance of {@link Handsontable}
+     *
+     * @private
+     * @type {Handsontable}
+     */
+    this.instance = instance;
+    /**
+     * Instance of {@link EventManager}
+     *
+     * @private
+     * @type {EventManager}
+     */
+    this.eventManager = new EventManager(instance);
+    /**
+     * Current Handsontable's GridSettings object.
+     *
+     * @private
+     * @type {GridSettings}
+     */
+    this.settings = instance.getSettings();
+    /**
+     * Main <THEAD> element.
+     *
+     * @type {HTMLTableSectionElement}
+     */
+    this.THEAD = void 0;
+    /**
+     * Main <TBODY> element.
+     *
+     * @type {HTMLTableSectionElement}
+     */
+    this.TBODY = void 0;
+    /**
+     * Main Walkontable instance.
+     *
+     * @type {Walkontable}
+     */
+    this.wt = void 0;
+
+    privatePool.set(this, {
+      /**
+       * Defines if the text should be selected during mousemove.
+       *
+       * @private
+       * @type {Boolean}
+       */
+      selectionMouseDown: false,
+      /**
+       * @private
+       * @type {Boolean}
+       */
+      mouseDown: void 0,
+      /**
+       * Main <TABLE> element.
+       *
+       * @private
+       * @type {HTMLTableElement}
+       */
+      table: void 0,
+      /**
+       * Main Walkontable instance.
+       *
+       * @private
+       * @type {Walkontable}
+       */
+      activeWt: void 0
+    });
+
+    this.createElements();
+    this.registerEvents();
+    this.initializeWalkOnTable();
```

Comment:
I would prefer using `this.initializeWalkontable();` name. The same practices are used for `handsontable` (not `handsOnTable`).